_includes/nav.html

```diff
@@ -65,20 +64,13 @@
               <div class="arrow down md theme_denim indicator"></div>
             </div>
             <div class="accordion-panel">
-              <a href="https://developer.confluent.io/learn-kafka/apache-kafka/events/#what-is-apache-kafka">What is
-                Apache Kafka®?</a>
-              <a href="https://developer.confluent.io/learn/event-streaming-vs-related-trends/">Event Streaming vs.
-                Related Trends</a>
-              <a href="https://developer.confluent.io/learn/kraft/">KRaft: Kafka without ZooKeeper</a>
-              <a href="https://developer.confluent.io/learn/kafka-transactions-and-guarantees/">Transactions &amp;
-                Guarantees in Kafka</a>
-              <a href="https://developer.confluent.io/learn/kafka-storage-and-processing/">Processing &amp; Storage
-                Fundamentals of Kafka</a>
-              <a href="https://developer.confluent.io/learn/kafka-performance/">Kafka Performance</a>
-              <a href="https://developer.confluent.io/learn/cloud-native-kafka/">Cloud-native Kafka</a>
-              <a href="https://developer.confluent.io/learn/streaming-database-systems/">Streaming Database Systems</a>
-              <a href="https://developer.confluent.io/learn/testing-kafka/">Testing Apache Kafka</a>
-              <a href="https://developer.confluent.io/#kafka-internals">Explore Kafka's Internals</a>
+              <a href="https://developer.confluent.io/learn/">Articles</a>
+              <a href="https://developer.confluent.io/patterns/">Patterns</a>
+              <a href="https://developer.confluent.io/learn/apache-kafka-faqs/">FAQs</a>
+              <a href="https://developer.confluent.io/100-days-of-code/">100 Days of Code</a>
+              <a href="https://www.confluent.io/blog/">Blog</a>
+              <a href="https://developer.confluent.io/podcast/">Podcast</a>
+              <a href="https://developer.confluent.io/live/">Confluent Developer Live</a>
```

Comment:
I don't see this link from the `Learn` menu on the KT site

Reply:
You mean on the production? It's a new menu item being added here.